app/routes/groups_.$groupId.tsx

```diff
@@ -8,7 +8,8 @@ import { Image } from '~/components/ui/images';
 
 export const loader: LoaderFunction = async ({ params }) => {
   const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
-  return json({ group });
+  const user_group = await db.user_Group.findMany({ where: { groupId: group.id } });
```

Comment:
```suggestion
  const user_groups = await db.user_Group.findMany({ where: { groupId: group.id } });
```

Reply:
Could be worth joining/including the user name for each user. Can be a follow-up!